### PR TITLE
change a bunch of descriptions in the Pause menu

### PR DESCRIPTION
**Pull request by hawkrives**
Also we should make a `menus` Github label. Or maybe `data`.

### CODE REVIEW

File: docs/menus/pause.json [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
+      "label": "Shakes",
+      "note": "Mixins: M&Ms, Oreos, Reese’s Peanut Butter Cups"
+    },
+    {
+      "label": "Soft-Serve",
+      "note": "Hand-swirled in a cup or a cone",
+    },
+    {
+      "label": "Pizza",
+      "note": "Includes a cup of Marinara, Ranch, BBQ, or Honey Mustard sauce"
```
Comment (drewvolz) [inline docs/menus/pause.json]: This is the note for Chicken Strips. Perhaps we do not need a note here.

File: docs/menus/pause.json [hunk 2]
```diff
@@ -100,39 +121,30 @@
...
       "station": "Quesadilla"
     },
     {
+      "description": "Pizza built on pita bread; includes two toppings",
       "label": "Pita Pizza",
       "special": true,
       "station": "Pita"
     },
     {
+      "description": "Pizza built on pita bread; includes two toppings",
```
Comment (drewvolz) [inline docs/menus/pause.json]: I do not think "specialty" items include an "extra two toppings". We need to check with a kitchen manager or worker to be sure.

**Review by drewvolz — CHANGES_REQUESTED** [review]

File: docs/menus/pause.json (hunk 1 above)
Comment (drewvolz) [reply docs/menus/pause.json]: On another note, large orders are a pain-point for the pause. We could use the note here to communicate "For orders of 4 or more pizzas, visit Oleville"

Comment (hawkrives) [reply docs/menus/pause.json]: Can do

**Comment by hawkrives** [comment]
@drewvolz I addressed your comments.

(I need to merge this soon, so my stationMenus stuff can be tested before I submit the filters PR)

**Review by drewvolz — APPROVED** [review]

**Pull request merged**